packages/babel-parser/test/fixtures/experimental/_no-plugin/numeric-separator-radix/options.json

```diff
@@ -1,4 +1,3 @@
 {
-  "throws": "This experimental syntax requires enabling the parser plugin: 'bigInt' (1:1)",
-  "plugins": []
-}
+  "throws": "This experimental syntax requires enabling the parser plugin: 'numericSeparator' (1:3)"
+}
```

Comment:
The diff here may be a bit confusing as I also remove the unused `bigint` missing plugin errors.